apps/charterafrica/src/lib/ecosystem/airtable/processData.js

```diff
@@ -244,7 +244,7 @@ export function processOrganisation(
     avatarUrl:
       getValue(data, organisationTableColumns.avatarUrl)?.[0]?.url ?? null,
     externalId,
-    type: getValue(data, organisationTableColumns.type),
+    type: getValue(data, organisationTableColumns.type)?.replace(/&/g, "and"),
```

Comment:
What is happening here?

Reply:
Related to [this](https://github.com/CodeForAfrica/ui/pull/554#discussion_r1302789188)